### PR TITLE
Start all services before running test scripts

### LINKED ISSUE #315: Start all services before running test script
Based on the discussion on [developer-script PR #372](https://github.com/edgexfoundry/developer-scripts/pull/372#issuecomment-765486501), update the scripts to start all services before running all tests.

### PR DESCRIPTION
**Pull request by cherrycl**
1. Start all services before running test scripts.
2. Update run-tests-on-local.md

Fix #315 
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

File: TAF/utils/scripts/docker/run-tests.sh [hunk 1]
```diff
@@ -41,22 +42,44 @@ fi
...
+              -v /var/run/docker.sock:/var/run/docker.sock ${TAF_COMMON_IMAGE} \
+              --exclude Skipped -u functionalTest/device-service -p device-virtual
+        cp ${WORK_DIR}/TAF/testArtifacts/reports/edgex/log.html ${WORK_DIR}/TAF/testArtifacts/reports/cp-edgex/virtual.html
+      ;;
+      device-modbus)
+        docker run --rm --network host --name taf-common -v ${WORK_DIR}:${WORK_DIR}:z -w ${WORK_DIR} \
+              --security-opt label:disable -e COMPOSE_IMAGE=${COMPOSE_IMAGE} -e ARCH=${ARCH} \
+              -e SECURITY_SERVICE_NEEDED=${SECURITY_SERVICE_NEEDED} \
+              -v /var/run/docker.sock:/var/run/docker.sock ${TAF_COMMON_IMAGE} \
+              --exclude Skipped -u functionalTest/device-service -p device-virtual
```
Comment (jinlinGuan) [inline TAF/utils/scripts/docker/run-tests.sh:59]:
```suggestion
              --exclude Skipped -u functionalTest/device-service -p device-modbus
```

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: TAF/utils/scripts/docker/run-tests.sh (hunk 1 above)
Comment (cherrycl) [reply TAF/utils/scripts/docker/run-tests.sh]: Fixed.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Comment by cherrycl** [comment]
Removed deploy service steps for integration-test and update configuration.toml for device-services.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Pull request merged**